<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [openssl-users] Self-signed error when using SSL_CTX_load_verify_locations CApath
   </TITLE>
   <LINK REL="Index" HREF="https://mta.openssl.org/pipermail/openssl-users/2018-December/index.html" >
   <LINK REL="made" HREF="mailto:openssl-users%40openssl.org?Subject=Re%3A%20%5Bopenssl-users%5D%20Self-signed%20error%20when%20using%0A%20SSL_CTX_load_verify_locations%20CApath&In-Reply-To=%3CA01B25EE-2BFE-44D1-8CE6-ED99CBA6C9A3%40dukhovni.org%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="009355.html">
   <LINK REL="Next"  HREF="009341.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[openssl-users] Self-signed error when using SSL_CTX_load_verify_locations CApath</H1>
    <B>Viktor Dukhovni</B> 
    <A HREF="mailto:openssl-users%40openssl.org?Subject=Re%3A%20%5Bopenssl-users%5D%20Self-signed%20error%20when%20using%0A%20SSL_CTX_load_verify_locations%20CApath&In-Reply-To=%3CA01B25EE-2BFE-44D1-8CE6-ED99CBA6C9A3%40dukhovni.org%3E"
       TITLE="[openssl-users] Self-signed error when using SSL_CTX_load_verify_locations CApath">openssl-users at dukhovni.org
       </A><BR>
    <I>Mon Dec  3 01:54:29 UTC 2018</I>
    <P><UL>
        <LI>Previous message: <A HREF="009355.html">[openssl-users] Self-signed error when using	SSL_CTX_load_verify_locations CApath
</A></li>
        <LI>Next message: <A HREF="009341.html">[openssl-users] [EXTERNAL] Re: Self-signed error when using SSL_CTX_load_verify_locations CApath
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#9358">[ date ]</a>
              <a href="thread.html#9358">[ thread ]</a>
              <a href="subject.html#9358">[ subject ]</a>
              <a href="author.html#9358">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>&gt;<i> On Dec 2, 2018, at 7:43 PM, Charles Mills &lt;<A HREF="../../../mailman/listinfo/openssl-users.html">charlesm at mcn.org</A>&gt; wrote:
</I>&gt;<i> 
</I>&gt;<i> Sorry, I do not have a packet capture tool configured.
</I>&gt;<i> 
</I>&gt;<i> I have a verify callback with a lot of trace messages. I can see that it is
</I>&gt;<i> only entered once; X509_STORE_CTX_get_error_depth() is 1.
</I>&gt;<i> 
</I>&gt;<i> Does that tell us anything useful?
</I>
No further information is required.  Your client certificate chain
includes a self-signed root CA as a direct issuer of its certificate.
That root CA was not found in the server's trust store.

Someone should submit a pull request to improve the error message, if
they've not done so yet.

-- 
-- 
	Viktor.

</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="009355.html">[openssl-users] Self-signed error when using	SSL_CTX_load_verify_locations CApath
</A></li>
	<LI>Next message: <A HREF="009341.html">[openssl-users] [EXTERNAL] Re: Self-signed error when using SSL_CTX_load_verify_locations CApath
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#9358">[ date ]</a>
              <a href="thread.html#9358">[ thread ]</a>
              <a href="subject.html#9358">[ subject ]</a>
              <a href="author.html#9358">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="../../../mailman/listinfo/openssl-users.html">More information about the openssl-users
mailing list</a><br>
</body></html>
